File: java/KeepUpTheHoop.java
// return an encouragement based on input value

public class KeepUpTheHoop {

  public static String hoopCount(int n) {
    return n >= 10 ? "Great, now move on to tricks" : "Keep at it until you get it";
  }

  public static void main(String[] args) {
    System.out.println(hoopCount(9)); // "Keep at it until you get it"
    System.out.println(hoopCount(10)); // "Great, now move on to tricks"
    System.out.println(hoopCount(11)); // "Great, now move on to tricks"
  }
}

File: c#/KeepUpTheHoop.cs
// return encouraging messages based on how many hoops

public class KeepUpTheHoop
{
	public static string HoopCount(int n)
	{
		if (n >= 10) return "Great, now move on to tricks";
		else return "Keep at it until you get it";
	}

	public static void Main(string[] args)
	{
		System.Console.WriteLine(HoopCount(9)); // keep at it
		System.Console.WriteLine(HoopCount(10)); // move on
		System.Console.WriteLine(HoopCount(11)); // move on
	}
}